apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx

```diff
@@ -169,7 +172,8 @@ export const SignInMethodLanding: FunctionComponent<SignInMethodLandingPropsInte
                                     } }
                                 />
                             ) }
-                            { !hiddenOptions.includes(LoginFlowTypes.SECOND_FACTOR_SMS_OTP) && (
+                            { (!hiddenOptions.includes(LoginFlowTypes.SECOND_FACTOR_SMS_OTP) &&
```

Comment:
Shall we move the condition to a newline and improve the formatting in this segment?
```suggestion
                            { 
                                (!hiddenOptions.includes(LoginFlowTypes.SECOND_FACTOR_SMS_OTP) &&
```